src/bp-core/admin/bp-core-admin-types.php

```diff
@@ -116,6 +140,8 @@ function bp_core_admin_insert_type( $args = array() ) {
 		return $type_term_id;
 	}
 
+	$type_term_id = reset( $type_term_id );
```

Comment:
This is currently an array containing the `term_id` and `term_taxonomy_id`, not the term id.